### ISSUE TITLE
false positives with schema.core/defmethod

### ISSUE THREAD

**Issue by mynomoto** [borkdude/clj-kondo#402]
**version**
clj-kondo v2019.07.31-alpha

**platform**
linux

**problem**
It fails to recognize arguments on a `schema.core/defmethod`

**repro**
```
(ns cljkondo
  (:require
    [schema.core :as sc]))

(sc/defmethod ig/init-key :config :- {:config/env sc/Keyword}
  [_
   {:keys [:config/env]} :- {:config/env sc/Keyword}]
  {:config/env env})
```

**expected behavior**
No warnings/infos

**actual behavior**
```
$ clj-kondo --lint cljkondo.clj 
cljkondo.clj:6:4: info: unresolved symbol _
cljkondo.clj:8:16: info: unresolved symbol env
linting took 10ms, errors: 0, warnings: 0
```

**Comment by borkdude** [comment]
@mynomoto This should be fixed using this binary:

https://5324-176829714-gh.circle-artifacts.com/0/release/clj-kondo-2019.08.01-alpha-SNAPSHOT-linux-amd64.zip

**Comment by mynomoto** [comment]
@borkdude I tested it and it works. Thank you very much!
